src/Razor/src/rzls/Endpoints/UpdateLogLevelEndpoint.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts;
+using Microsoft.CodeAnalysis.Razor.Logging;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Endpoints;
+
+internal class UpdateLogLevelEndpoint(LogLevelProvider logLevelProvider) : IRazorNotificationHandler<UpdateLogLevelParams>
```

Comment:
I was on the fence of where to put this. I realize why I haven't put other rzls endpoints here: we don't have a testing suite for these endpoints if they are defined here. 

This one is fairly simple so I didn't add tests but I could be swayed to move this to LanguageServer if anyone cares enough

Reply:
If you do end up moving this to LanguageServer, stick it in the Hosting folder or a nested folder within.